tyk-data-plane/Chart.yaml

```diff
@@ -16,7 +16,19 @@ type: application
 # to the chart and its templates, including the app version.
 # Versions are expected to follow Semantic Versioning (https://semver.org/)
 version: 1.0.0
-
+home: https://tyk.io/
+sources:
+  - https://github.com/TykTechnologies/tyk-charts
+  - https://community.tyk.io/
+  - https://tyk.io/docs/
```

Comment:
please update source URL to exact chart URL
please move other links to annotations